packages/hasura/config.yaml

```diff
@@ -1,5 +1,5 @@
 version: 3
-endpoint: http://localhost:8080
+endpoint: https://hasura-staging-d83b.up.railway.app
```

Comment:
Is this correct or should we now start to have staging vs production branches?

Reply:
yeah maybe we need to remove the explicit endpoint configuration, forgot I had added that.

Reply:
actually I think we should keep this value as is. This is a required value, and localhost:8080 is the default. Can't really have environment specifics in this here.

Reply:
I see you using env vars in other places, should that not be what we do or am I missing something?